wtr-utils.js

```diff
@@ -61,12 +85,14 @@ const getUnitTestPackages = () => {
 const getVisualTestPackages = () => {
   // If --group flag is passed, return all packages.
   if (group) {
-    return getAllPackages();
+    return getAllVisualPackages();
   }
 
   let packages = getChangedPackages()
     .map((project) => project.name.replace('@vaadin/', ''))
-    .filter((project) => NO_UNIT_TESTS.indexOf(project) === -1 && project.indexOf('mixin') === -1);
+    .filter((project) => NO_UNIT_TESTS.indexOf(project) === -1 && project.indexOf('mixin') === -1)
+    .concat(getUpdatedScreenshotsPackages())
+    .filter((v, i, a) => a.indexOf(v) === i);
```

Comment:
Looking at this line, it took a time before I've got the idea behind this `filter`.

Since we don't have a `unique` method, I would appreciate a note that it returns an array of unique projects.

Reply:
Good point, thanks. Updated the script to make this clear.

Reply:
Is it possible for `getChangedPackages()` and `getUpdatedScreenshotsPackages()` to intersect each other in some cases?

If so, then now using `Set` in `getUpdatedScreenshotsPackages` seems not to solve the problem, so that the `packages` variable can contain non-unique values, I guess.